spacy_llm/pipeline/llm.py

```diff
@@ -1,11 +1,23 @@
 from pathlib import Path
-from typing import Dict, Iterable, Iterator, List, Optional, Tuple, Union, cast
+from typing import (
+    Any,
+    Callable,
+    Dict,
+    Iterable,
+    Iterator,
+    List,
+    Optional,
+    Tuple,
+    Union,
+    cast,
+)
```

Comment:
Minor: in line with spaCy we don't want imports to be broken up into multiple lines (seperate it into n lines if necessary).

Reply:
True. I'll modify the pre-commit configuration in another PR, we've come up with a clean solution for that in Weasel.